indico/modules/events/registration/models/items.py

```diff
@@ -437,6 +437,12 @@ def view_data(self):
     def _get_default_log_data(self):
         return {'Section ID': self.id}
 
+    @make_interceptable
+    def can_see(self, user, is_management):
+        if not user and self.registration_form.require_login:
+            return False
+        return not self.is_manager_only or is_management
```

Comment:
I think it's easier to read that way:

```suggestion
        return is_management or not self.is_manager_only
```